lib/rules/prop-name-casing.js

```diff
@@ -46,7 +56,12 @@ function create (context) {
           data: {
             name: propName,
             caseType: caseType
-          }
+          },
+          fix: canFixPropertyName(item) ? fixer => {
+            return item.key.type === 'Literal'
+              ? fixer.replaceText(item.key, item.key.raw.replace(item.key.value, convertedName))
```

Comment:
If the original name is `"🍻"`, what does it happen?

Reply:
Why!? It became "" !!!
Emoji is difficult...